tests/CodeSnifferRunner.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace ObjectCalisthenics\Tests;
+
+use PHP_CodeSniffer;
+
+final class CodeSnifferRunner
+{
+    /**
+     * @var PHP_CodeSniffer
+     */
+    private $codeSniffer;
+
+    public function __construct()
+    {
+        $this->codeSniffer = new PHP_CodeSniffer();
+    }
+
+    /**
+     * @param string $testedFile
+     * @param string $sniffName
+     *
+     * @return int
+     */
+    public function detectErrorCountInFileForSniff($testedFile, $sniffName)
+    {
+        $this->codeSniffer->initStandard('../../../../../src/ObjectCalisthenics', [$sniffName]);
```

Comment:
It's due to PHP_CodeSniffer, it's very bad at autoloading and relative/absolute paths mixture.
Here it should lead to directory with `ruleset.xml`.

Other options like name failed for me, because again, it requires specific placement of files.

Any suggestion?